docs/api/latest.md

```diff
@@ -1,4 +1,4 @@
-# API Docs - v4.0.16
+# API Docs - v4.0.17-SNAPSHOT
```

Comment:
Do not commit this file, it is auto generated

Reply:
Fixed with commit : 8256daa2540843332ceb13479eafbc9a6a4f5865